src/app/tracing/dialog/station-properties/station-properties.component.ts

```diff
@@ -634,6 +636,18 @@ export class StationPropertiesComponent implements OnInit, OnDestroy {
                     ),
                 );
         };
+
+        const updateToolTip = (node: NodeDatum, incoming: boolean): string => {
+            const { lot, name, station, date } = node;
+
+            const line1 = lot ? name + " (" + lot + ")" : (name ?? "");
+            const line2Prefix = incoming ? "from:" : "to:";
+            const line2 = station ? `${line2Prefix} ${station}` : "";
+            const line3 = date ?? "";
+
```

Comment:
The text assembly for the lines in tooltip and boxes is redundant. Please remove code duplication.

Reply:
- [ ] create func that provides text to both